frontend/src/pages/destructionlist/create/DestructionListCreate.tsx

```diff
@@ -34,41 +34,37 @@ export type DestructionListCreateContext = {
   selectedZaken: Zaak[];
 };
 
-export const getZakenData = async (
-  request: Request,
-  searchParamsZakenEndpoint: Record<string, string>,
-) => {
-  const searchParams = new URL(request.url).searchParams;
-  Object.keys(searchParamsZakenEndpoint).forEach((key) =>
-    searchParams.set(key, searchParamsZakenEndpoint[key]),
-  );
-
-  // Get reviewers, zaken and zaaktypen.
-  const promises = [listReviewers(), listZaken(searchParams)];
-  const [reviewers, zaken] = (await Promise.all(promises)) as [
-    User[],
-    PaginatedZaken,
-  ];
-
-  // Get zaak selection.
-  const isZaakSelectedPromises = zaken.results.map((zaak) =>
-    isZaakSelected(DESTRUCTION_LIST_CREATE_KEY, zaak),
-  );
-  const isZaakSelectedResults = await Promise.all(isZaakSelectedPromises);
-  const selectedZaken = zaken.results.filter(
-    (_, index) => isZaakSelectedResults[index],
-  );
-
-  return { reviewers, zaken, selectedZaken };
-};
-
 /**
  * React Router loader.
  * @param request
  */
 export const destructionListCreateLoader = loginRequired(
   async ({ request }: LoaderFunctionArgs) => {
-    return await getZakenData(request, { not_in_destruction_list: "true" });
+    const searchParamsZakenEndpoint: Record<string, string> = {
+      not_in_destruction_list: "true",
+    };
+    const searchParams = new URL(request.url).searchParams;
+    Object.keys(searchParamsZakenEndpoint).forEach((key) =>
+      searchParams.set(key, searchParamsZakenEndpoint[key]),
+    );
+
+    // Get reviewers, zaken and zaaktypen.
+    const promises = [listReviewers(), listZaken(searchParams)];
+    const [reviewers, zaken] = (await Promise.all(promises)) as [
```

Comment:
Since we said that for now the reviewers are assigned automatically when the destruction list is created, it should not be possible to update the reviewers. I need to also remove this from the backend :thinking:

Reply:
As discussed: we'll leave it in for now, possible remove/alter it later.